enhancements/two-node-fencing/tnf.md

```diff
@@ -193,9 +193,20 @@ See the API Extensions section below for sample install-configs.
 
 For a two-node cluster to be successful, we need to ensure the following:
 1. The BMC secrets for RHEL-HA are created on disk during bootstrapping by the OpenShift installer via a MachineConfig.
-2. When pacemaker is initialized by the in-cluster entity responsible for starting pacemaker, pacemaker will try to set up fencing with this secret. If this is not successful, it throws an error and the installation fails.
-3. Pacemaker periodically checks that the fencing agent is healthy (i.e. can connect to the BMC) and throws a warning if it cannot access the BMC. There is an open question on what the user experience should be to raise this error to the user.
-4. The cluster will continue to run normally in the state where the BMC cannot be accessed, but ignoring this warning will mean that pacemaker can only provide a best-effort recovery - so operations that require fencing will need manual recovery.
+2. When pacemaker is initialized by the in-cluster entity responsible for starting pacemaker, pacemaker will try to set up fencing with this secret. If this is not successful, it throws an error which will cause degradation of the in cluster operator and would fail the installation process.
+3. Pacemaker periodically checks that the fencing agent is healthy (i.e. can connect to the BMC) and will create an alert if it cannot access the BMC.
+   * In this case, in order to allow a simple manual recovery by the user a script will be available on the node which will reset Pacemaker with the new fencing credentials.
+4. The cluster will continue to run normally in the state where the BMC cannot be accessed, but ignoring this alert will mean that pacemaker can only provide a best-effort recovery - so operations that require fencing will need manual recovery.
+5. When manual recovery is triggered by running the designated script on the node, it'll also update the Secret in order to make sure the Secret is aligned with the credentials kept in Pacemaker's cib file.
```

Comment:
Again, two different things. Updating the credentials means its a delayed but automated recovery.
Manual recovery is when no fencing occurred and the user instructs the cluster to continue anyway.